linode/helper/frameworkfilter/local_filter.go

```diff
@@ -11,6 +11,8 @@ import (
 	"github.com/hashicorp/terraform-plugin-framework/types"
 )
 
+const EXACT = "exact"
```

Comment:
I think it's a little bit weird to only make `"exact"` a constant while leaving others (e.g., `"substring"`) unchanged. Maybe let's do this another PR.